### ISSUE TITLE
Analyze Training Loss/Eval Graphs

### ISSUE THREAD

**Issue by AndrewOlvera-byte** [YCP-Rev-Metrix/Ciclopes#10]
- [ ] Overfitting? Underfitting?
